src/lib/helpers/CameraHelper.svelte

```diff
@@ -0,0 +1,20 @@
+<script lang="ts">
+    import { CameraHelper, Camera } from 'three'
+    import Object3DInstance from '../instances/Object3DInstance.svelte'
+    import { getParent } from '../internal/HierarchicalObject.svelte'
+  
+    const parent = getParent()
+  
+    let helper: CameraHelper
+  
+    if (!($parent instanceof Camera)) {
+      console.warn('<CameraHelper> can only be used as a direct child of <PerspectiveCamera>, <OrthographicCamera> ') //This description is somewhat innacurate and needs to be improved once <StereoCamera> is implemented.
+    } else {
+      helper = new CameraHelper($parent)
+    }

```

Comment:
The `CameraHelper`s method [`dispose`](https://github.com/mrdoob/three.js/blob/860af3c012915be534f8a5c37d745e2dff339e04/src/helpers/CameraHelper.js#L180) should be called `onDestroy`.